### ISSUE TITLE
Workshop preparation

### ISSUE THREAD

**Issue by annefou** [annefou/jupyter_dashboards#1]
Just an overall question to finalize our workshops.

- Do we need to add/change anything for ipython widgets section?
- environment.yml needs to be updated for beakerx (I think Gladys did it)
- We still need to add an example on how to use Binder with a Dockerfile, isn't it?
- As soon as I get the list of attendees, I'll send an email (template given by Research Bazaar + add info on our setup)
- Any comments/inputs, missing topics?

**Comment by GladysNalvarte** [comment]
@annefou  
For environment.yml : Yes, I have updated the environment.yml file. I have included ipywidgets, beakers and folium.
For the example with Dockerfile: In the binder documentation (https://mybinder.readthedocs.io/en/latest/dockerfile.html) is mentioned that it is not recommended to use Dockerfiles with Binder it is not guarantee that the Dockerfile will work. 
I suggest that we can present some possible issues that can be expected to be found when we work with Binder and the possible workarounds.
